extension/compile.js

```diff
@@ -0,0 +1,22 @@
+const fs = require("fs");
+const { Liquid } = require("liquidjs");
+const version = require("./package.json").version;
+
+const FILES = [
+  "./manifest.json",
+  "./popup.html",
+  "./js/background.js"
+];
+
+const engine = new Liquid({
+  globals: {
+    host: process.env.HOST || "http://127.0.0.1",
+    port: process.env.PORT || "",
```

Comment:
I figured these were easier as ENV vars. But with minimal fiddling I could switch to CLI args if you prefer. It's look more like this down in package.json scripts below `node compile --host=https://relay.firefox.com` or `node compile --port=":8000"` (versus `HOST=https://relay.firefox.com node compile`).